thirtyinch/src/main/java/net/grandcentrix/thirtyinch/internal/DelegatedTiFragment.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2016 grandcentrix GmbH
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.grandcentrix.thirtyinch.internal;
+
+import android.app.Activity;
+import android.support.v4.app.Fragment;
+
+public interface DelegatedTiFragment {
```

Comment:
> some methods overlap

Currently only the `P getRetainedPresenter();` don't overlap. The question for me is: Why we haven't it inside the fragment? 

> I don't see a reason for inheritance